js/components/arrowButtonMaker.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Creates a button of a specific type.
+ * @param {object} props - The properties of the arrow button.
+ * @param {string} props.type - The type of button ("left" or "right").
+ * @param {int} props.index - We're passing the current fighter step index for onClick to work.
+ * @returns {HTMLElement} - The generated button element.
+ * @throws {Error} - Throws an error if the type is not "left" or "right".
+ */
+
+import { chooseFighterStep } from '../wizard/steps/choose-fighter-step/choose-fighter-step.js';
+
+export function arrowButtonMaker(props) {
+   const type = props.type;
+   const index = props.index;
+
+   if (type !== 'left' && type !== 'right') {
+      throw new Error('Invalid button type. Only "left" and "right" are allowed.');
+   }
+
+   let button = document.createElement('button');
+   if (type == 'right') {
+      button.style.backgroundImage = 'url(../assets/buttonNext.svg)';
```

Comment:
**nitpick:** Then you could just remove these ifs - you don't need them

Reply:
I'll apply your comments, but I've gotta say that's too much generalisation for my taste. Now basically the only difference between bigButtonMaker and arrowButtonMaker is the button's size - which now begs the question what was the point of distinguishing between those two in the first place? Why not make one buttonMaker function that allows me to create an arbitrary button with a huge props object every time?

Reply:
Making it as one is also fine - my main focus is that you should not have to make ifs in such simple logic. If you can parametrize these two buttons the way it has one common Make function - go for it:)

Reply:
That's the thing - I could, but then I'd have to establish the buttons' styles & properties every time I create such button. I thought the idea behind making a button-making function was that I could "hide" all those messy, single-liners such as multiple "button.style.height = '123px'" etc...

Why not just make document.createElement('button') every time?